test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ClientYamlTestResponse.java

```diff
@@ -71,6 +72,14 @@ public ClientYamlTestResponse(Response response) throws IOException {
         }
     }
 
+    private XContentType getContentTypeIgnoreExceptions(String contentType) {
```

Comment:
I'd love if we can move the comment from the usage to a method level documentation that expands on why we use null; I get it now but at some point it won't be obvious anymore

Reply:
true, will add more comment on per method javadoc